src/plugins/htmlAudioPlayer/plugin.js

```diff
@@ -1,91 +1,94 @@
-define(['events', 'browser', 'require', 'apphost', 'appSettings', 'htmlMediaHelper'], function (events, browser, require, appHost, appSettings, htmlMediaHelper) {
-    'use strict';
-
-    function getDefaultProfile() {
-        return new Promise(function (resolve, reject) {
-            require(['browserdeviceprofile'], function (profileBuilder) {
-                resolve(profileBuilder({}));
-            });
+import events from 'events';
+import browser from 'browser';
+import appHost from 'apphost';
+import * as htmlMediaHelper from 'htmlMediaHelper';
+
+function getDefaultProfile() {
+    return new Promise(function (resolve, reject) {
+        import('browserdeviceprofile').then(({default: profileBuilder}) => {
+            resolve(profileBuilder({}));
         });
+    });
```

Comment:
I believe we could remove top `Promise` and do everything with `import`. Leave it for "Promise Cleanup" PR?

Reply:
I've removed the promise